templates/experiment.html

```diff
@@ -1,9 +1,9 @@
 <div id="experimental-mode" class="card info">
   <h4>Experimental Mode</h4>
-  <p>The <code>{{component}}</code> component is currently in experimental mode and needs to be manually enabled (<a href="https://www.ampproject.org/docs/reference/experimental.html">more info</a>).</p>
+  <p>This example uses the following experimental feature{{#metadata.experiments.1}}s{{/metadata.experiments.1}}: <code class="amp-experiment-list">[{{#metadata.experiments}}<span>{{.}}</span>{{/metadata.experiments}}]</code><br>Enable them below and <a href="https://www.ampproject.org/docs/reference/experimental.html">learn more here</a>.</p>
```

Comment:
We are doing the check for the plural ({{#metadata.experiments.1}}s{{/metadata.experiments.1}}) but we are not doing it for "Enable them". If it's too complex check everytime before write the string, then using the array style would be better, otherwise let's be consistent.